lib/tasks/emailer.rake

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+
+namespace :emailer do
+  desc 'email task runner'
+  task vouchers_to_merchants: :environment do
+    desc 'sends all voucher codes to all of their merchants'
+
+    s_sql = 'select c.email as seller_contact_email,
```

Comment:
oh wow any reason why you used sql instead of ActiveRecord?

Reply:
This one could most likely be converted to ActiveRecord--I started with the other query first and it seemed more complex than I could get through ActiveRecord, but maybe I'll give it a shot. I'm not the hugest fan of all this raw sql here, but it was a good place for me to start as I figure I probably know that better than the ActiveRecord API 😬

Reply:
ah makes sense. active record is nice because it has checks and stuff built into the models. happy to pair with you on this!

Reply:
Sounds good to me! Let me take another pass at it, then would be great to pair sometime this weekend!